crates/bevy_ecs/src/world/entity_ref.rs

```diff
@@ -70,6 +70,13 @@ impl<'w> EntityRef<'w> {
         }
     }
 
+    pub fn get_by_id(&self, component_id: ComponentId) -> Option<*const ()> {
```

Comment:
Doc comment please. I don't want ordinary users mucking around with this by mistake.

Reply:
Ditto for below of course.